packages/edit-site/src/components/experiments-page/index.js

```diff
@@ -0,0 +1,284 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	Button,
+	Spinner,
+	__experimentalConfirmDialog as ConfirmDialog,
+	__experimentalHStack as HStack,
+} from '@wordpress/components';
+import { DataForm } from '@wordpress/dataviews';
+import { __, _x } from '@wordpress/i18n';
+import { useEntityRecord } from '@wordpress/core-data';
+import { useMemo, useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { Page } from '@wordpress/admin-ui';
+import { useExperiments } from '../experiments-context';
+
+/**
+ * Group labels for experiment categories.
+ */
+const GROUP_LABELS = {
+	blocks: _x( 'Blocks', 'experiment group' ),
+	media: _x( 'Media', 'experiment group' ),
+	collaboration: _x( 'Collaboration', 'experiment group' ),
+	'data-views': _x( 'Data Views', 'experiment group' ),
+	interactivity: _x( 'Interactivity', 'experiment group' ),
+	'content-only': _x( 'contentOnly', 'experiment group' ),
+	templates: _x( 'Templates', 'experiment group' ),
+	other: _x( 'Other', 'experiment group' ),
+};
+
+/**
+ * Order in which groups should appear.
+ */
```

Comment:
I know you're not done, but just wondering it there's be anyway to sort/filter the dataviews list in the future.

Having a lot of experiments isn't the ideal state, but it's where we are, and it might make discovery easier?